Makefile

```diff
@@ -7,6 +7,7 @@ install-dev-rs-release:
 	cd sqlglotrs/ && python -m maturin develop -r
 
 install-dev-rs:
+	@unset CONDA_PREFIX && \
```

Comment:
I had to add this, as I had a conda prefix set, it's not totally uncommon for makefiles to set this to make things easier. For example, Polars does this: https://github.com/pola-rs/polars/blob/main/Makefile

Let me know if we're fine with keeping it.

Reply:
as long as this doesn't break my setup

Reply:
It shouldn't! haha